WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -561,6 +564,51 @@ public void hidePhotoChooser() {
             mPhotoChooserFragment.finishActionMode();
             AniUtils.animateBottomBar(mPhotoChooserContainer, false);
         }
+
+        View overlay = findViewById(R.id.view_overlay);
+        if (overlay.getVisibility() == View.VISIBLE) {
+            AniUtils.fadeOut(overlay, AniUtils.Duration.MEDIUM);
+        }
+    }
+
+    /*
+     * called by PhotoChooserFragment when user selects a single media uri
+     */
+    @Override
+    public void onPhotoChooserMediaChosen(@NonNull Uri uri) {
```

Comment:
Is there a reason to have this? a particular case of  `onPhotoChooserMediaListChosen(@NonNull List<Uri> uriList)` - if not it seems it could all be handled by  this latter one and get a list of one item, maybe renaming it to `onPhotoChooserMediaChosen(@NonNull List<Uri>)`. Unless you feel strong about having a separate method for one item selections.